docs/scripts.rst

```diff
@@ -365,6 +377,22 @@ These values are stored in the ``fields`` array of the ``schema`` dict of the JS
   year, int
   ...
 
+
+where ``name`` represents name of the column and ``type`` represents the type of data present in the column. Following can be used to describe the data type:
+
+::
+
+  pk-auto: Auto generated primary key starting from 1
+  pk-[char,int,double]: primary key with data type
+  char: strings
+  int: integers
+  double:floats/decimals
+  ct-[int,double,char]:Cross tab data
+  skip: used to skip the column in database
+
+
+``pk-auto`` is used to create an additional column of int data type which act as a primary key with values starting from 1. While ``pk-[char,int,double]`` is used make a primary key from existing columns of the table having data type of char/int/double.
```

Comment:
.....is used to create an additional column of **type int** which act**s** as a primary key with values starting from 1. While ``pk-[char,int,double]`` is used **to** make a primary key from existing columns of the table having data type of char/int/double.